src/Jackett.Common/Definitions/soundpark.yml

```diff
@@ -38,9 +38,9 @@
 
   search:
     paths:
-      - path: "{{if .Keywords}}search{{else}}music{{end}}"
+      - path: "search"
     inputs:
-      q: "{{if .Query.Artist}}{{ .Query.Artist }}{{else}}{{ .Keywords }}{{end}}"
+      q: "{{if or (.Query.Album) (.Query.Artist) }}{{ or (.Query.Album) (.Query.Artist) }}{{else}}{{ .Keywords }}{{end}}"
```

Comment:
we still need to support path music for when lidarr users add the indexer for the first time and/or do a TEST (which as far as I believe lidarr just sends a keyword-less query to jackett), as well as users that are using the jackett dashboard to do a search without keywords.
does the and/or support allow for coding a path that is the equivalent of this non-literal example:
-path "{{if .Keywords or .Query.Artist or .Query.Album}}search{{else}}music{{end}}" ?